systems/console-bff/common/configs/index.ts

```diff
@@ -20,7 +20,7 @@ export const AUTH_APP_URL = process.env.AUTH_APP_URL ?? "http://localhost:4455";
 export const PLAYGROUND_URL =
   process.env.PLAYGROUND_URL ?? "http://localhost:8080";
 export const CONSOLE_APP_URL =
-  process.env.CONSOLE_APP_URL ?? "http://localhost:3000";
+  process.env.CONSOLE_APP_URL ?? "http://localhost:3001";
```

Comment:
Why we need to change port?

Reply:
I was testing, so I had to change the port since Lago API was running on the same one.